#+options:    H:7 num:2 toc:t \n:nil @:t ::nil |:t ^:{} -:t f:t *:t TeX:t LaTeX:t skip:t d:(HIDE) tags:not-in-toc
#+STARTUP:    align overview nodlcheck hidestars oddeven lognotestate latexpreview
#+SETUPFILE: https://fniessen.github.io/org-html-themes/org/theme-readtheorg.setup
#+SEQ_TODO:   NEXT(n) TODO(t) INPROGRESS(i) WAITING(w@) | DONE(d) CANCELED(c@)
#+TAGS:       Write(w) Update(u) Fix(f) Check(c) noexport(n) Aaron(a) Sam(s)
#+PROPERTY: header-args:R  :session *R_gen* :tangle yes
#+TITLE:    Spatial Statistics on Spatial Transcriptomics
#+AUTHOR:    Aaron Osgood-Zimmerman
#+EMAIL:     aaron.oz@bucknell.edu
#+DESCRIPTION: Stats ideas, comments, and notes for spatial transcriptomics data and problems. Collab with MS Raredon
#+LANGUAGE:   en
#+csl-style: ./nature.csl
#+LATEX_HEADER: \usepackage{amsmath}
#+LATEX_HEADER: \usepackage[margin=0.75in]{geometry}
#+LATEX_HEADER: \usepackage{parskip}
#+LATEX_HEADER: \usepackage{natbib}
#+latex_header: \usepackage{dsfont}
#+BIND: org-html-table-caption-above nil

* Intro
Here are some (naive) and v1 attempts to think about how spatial stats
tools might be used. I read /Museum of spatial transcriptomics/[[cite:&moses2022museum]]
section on data analysis methods after writing most of this and
most/all of the ideas I toss out seem to already have a use in the
field for some application.

I think it might be more fruitful to start with what problems we want
to solve and then work from there, but giving Sam some better idea of
spatial stats tools might help define the problem.

Still, I think we should meet again and talk about what question Sam
most dreams of answering and figuring out what might be done to get us
(closer to) there.

* Useful resources
** Resources to consume
   - [ ] [[https://www.nature.com/articles/s41588-023-01435-6][Spatiotemporal transcriptomic maps of whole mouse embryos at the onset of organogenesis]]
   - [ ] [[https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02653-7][Statistical and machine learning methods for spatially resolved transcriptomics data analysis]]
   - [ ] [[https://pubmed.ncbi.nlm.nih.gov/36452868/][Spatial statistics is a comprehensive tool for quantifying cell neighbor relationships and biological processes via tissue image analysis]]
   - [ ] [[https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8494229/][Advances in spatial transcriptomic data analysis]]

** Spatial Statistics
    - Bayesian melding of different resolutions[[cite:&fuentes2005]]

** Spatial Transcriptomics
    - [[https://www.nature.com/articles/s41592-022-01409-2][Museum of spatial transcriptomics]][[cite:&moses2022museum]]

      Broad description of methods being used and the associated problems they address:

      #+begin_quote
      Data Analysis - Downstream

      Downstream analyses most often apply to the gene count matrix and cell or spot locations, and are thus largely independent of data-collection technologies.

Given the relevance of scRNA-seq to spatial data, and how spatial data
are often analyzed like scRNA-seq data in exploratory data analysis
(EDA), popular scRNA-seq EDA ecosystems, such as Seurat3, SCANPY
(which spatical single-cell analysis in Python (Squidpy) is built
on), and SingleCellExperiment (extended by SpatialExperiment),
have added functionalities for spatial data, such as updates to data
containers and functions to facilitate visualization of gene
expression and cell or spot metadata at spatial locations (Section 7.2
of Supplementary Information). EDA packages dedicated to spatial data
with beautiful graphics and good documentation have also been written,
such as Giotto and STUtility. Seurat and Giotto also implement
basic methods to identify spatially variable genes. In addition,
Giotto implements methods to identify cell-type enrichment in ST and
Visium spots, to identify gene coexpression and association between
gene expression and cell-type colocalization, and to identify spatial
regions.

Spatially variable genes are genes whose expression is associated with
spatial location (Section 7.5 of Supplementary Information). Three
approaches are commonly used for these genes: Gaussian process
regression (GPR) and its generalization to Poisson and N,
Laplacian score, and Moran’s I. GPR-based methods model normalize
gene expression or the rate parameter of Poisson or NB gene expression
as a GPR and find whether the model better describes the data with the
spatial term than without. Laplacian-score-based methods identify
genes whose expression better reflects the structure of a spatial
neighborhood graph. The locations of cells can also be modeled as a
spatial point process with gene expression as marks; spatially
variable genes can be identified as marks associated with
locations. Fitting GPR models to numerous genes can be time
consuming, especially when a Bayesian approach with Markov chain Monte
Carlo is used. Permutation testing used in Laplacian-score-based
methods can also be time consuming. As both GPR- and
Laplacian-score-based methods seek to identify spatial
autocorrelation, sometimes the classic spatial autocorrelation metric
Moran’s I is directly used to identify spatially variable genes, as in
Seurat v3 and above. MERINGUE uses a local version of Moran’s
I. Moran’s I and its significance testing are implemented in
established geospatial packages and are easy and fast to run, but may
have less statistical power than model-based methods.

Spatial information also enables identification of potential cell–cell
interactions (Supplementary Section 7.8). This is commonly done with
knowledge of ligand–receptor (L–R) pairs, and can test which L–R pairs
are more likely to be expressed in neighboring cells or spots or
whether two cell types each expressing the ligand and the receptor are
more likely to colocalize. The cross-type L function from a spatial
point process can be used to find cell types that
colocalize. Expression of genes of interest can also be modeled,
including a term for cell–cell colocalization; a gene is considered
associated with cell–cell colocalization if the model better describes
the data with this term than without.

There are many other types of downstream analysis that are useful for
spatial transcriptomics analysis, including identification of
archetypal gene patterns (Section 7.6 of Supplementary Information),
spatial regions defined by the transcriptome (Section 7.7 of
Supplementary Information), inferring gene–gene interactions (Section
7.9 of Supplementary Information), subcellular transcript localization
(Section 7.10 of Supplementary Information), and gene expression
imputation from H&E images (Section 7.11 of Supplementary
Information).
      #+end_quote
* Big Ideas in Spatial Stats and their (maybe) relevance
** Point process models
    Point process models deal with the pattern of points in a bounded
    space. A classic example being observed locations of a specific
    series of trees within a forest. The measured points may have
    associated data (tree diam, height, etc) that might be of interest to model.

    These models can be extended to handle /multiple types of points/.

*** Inferring cell types at spatial locations :IDEA:
    This might be useful if we're interested in using spatial
    transcriptomics to better map the location of cell types. For
    example, we might be able to use coarser spatial data to estimate
    the presence/absence of cell types at the different spatially
    observed grids.

** Spatial Downscaling

    There are different possible topical applications, but the main
    idea would be to use developed spatial statistics models that
    acknowledge and account for coarser resolution(s) of the data
    observations to model a finer resolution latent (unobserved)
    truth.

    Examples include coarse satellite measurements used to
    model finer resolution weather/climate, estimating county-level
    properties from state-level measurements, etc.

    This has very a very clear, understandable, and interpretable
    analogy to spatial transcriptomic data where the measurements are
    coarser than the single cells that we would like to directly
    model.

    Much of the downscaling literature is devoted to developing
    methods for comparing, correcting, or calibrating numerical model
    output using observed data from physical experiments and
    monitoring stations.

    Unless we have good(?) models of the cell (systems) you'd like to
    assess, this approach is probably not that useful to us. More
    likely relevant are approaches[[cite:&fuentes2005]] that address the
    change of support problem (small/fine scale truth mismatched to
    large/coarse scale observations) by directly expressing the
    coarse-resolution output as the integral/average over
    high-resolution grid cells, and build models using observations as
    ground truth.

    Below I describe some downscaling use cases in the spatial stats lit.

*** Downscaling via integration

     Fuentes and Raftery[[cite:&fuentes2005]] synthesize data from two
     different resolutions -- coarse but complete spatial coverage and
     high quality but spatially limited point sources -- to create a
     coherent model for the latent unobserved truth.

**** Better estimation by combining single cell and spatial data :IDEA:
     In some ways this is very similar to augmenting the spatial
     transcriptomics data with single cell data, except that we don't
     have any measurements about the spatial locations of the
     high-quality single cell data. This /maybe/ could be overcome use
     some data augmentation scheme to simultaneously estimate the
     spatial positions of the single cell data records and the
     underlying spatial field.

*** Observing system simulation experiments (OSSE)

#+begin_quote
Observing system simulation experiments (OSSEs) are widely used in
atmospheric studies and climate monitoring to guide development of new
observing systems ... and to evaluate performance of new data
assimilation algorithms In an OSSE, a simulated atmospheric field from
a numerical model is used as the "truth" to produce synthetic
observations by adding suitable measurement errors and other
representative errors. These synthetic observations are then fed to a
data assimilation algorithm. Here, data assimilation refers to the
process of fusing [coarse] observations from the observing systems
such as satellites together with numerical model output [from a model
of the system], to infer the true state of geophysical processes; see
Wikle and Berliner (2007) for a formal definition from a statistical
perspective.  Since OSSEs deal entirely with simulations, they provide
a cost-effective approach to evaluating the impact of new observing
systems and performance of new data assimilation algorithms. In
particular, OSSEs can be employed to compare competing observing
system designs. cite:&ma2019
#+end_quote


**** Assessment/comparison of novel/existing spatial collection techniques :IDEA:
Something like this could be used to assess different spatial data
collection techniques or to assess the improvements that a novel
spatial measuring technique could provide. Not something that I'm super interested in...



* Bibliography

[[bibliography:~/Dropbox/Genetics/spatial-transcriptome/spatial-transcriptome-lib.bib]]
